src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -318,7 +320,7 @@ AirspeedModule::Run()
 
 			/* poll airspeed data */
 			airspeed_s airspeed_raw = {};
-			orb_copy(ORB_ID(airspeed), _airspeed_sub[i], &airspeed_raw); // poll raw airspeed topic of the i-th sensor
+			_airspeed_sub[i].copy(&airspeed_raw); // poll raw airspeed topic of the i-th sensor
```

Comment:
If you stop getting airspeed data for any reason you might want to think about catching it here rather than propagating the same stale data to the airspeed validator instance.